client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,121 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support__wrapper space-inset-m">
+    <h2 class="font-normal">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <section class="mb-5">
+      <p class="mb-5">
+        {{ Translator.trans('support.introduction') }}
+      </p>
+      <p>
+        Sie haben eine Frage zur Technik? Dann hilft Ihnen unser technischer Support gerne weiter.
```

Comment:
If you use trans keys in other places in this component, I think it makes sense to pass this text with the trans key as well